### ISSUE TITLE
Bodymovin. How to minimize banner size?

### ISSUE THREAD

**Issue by parashut1986** [airbnb/lottie-web#2674]
Hi! Tell me, are there any tricks to minimize the final weight of the banner?
I have an animation for 8 seconds. The weight should be 150kb. The animation is done using a script. The banner with it weighs 177kb. Translated all expressions into keys, deleted extra keys and it turned out 197kb. Is it possible to reduce the weight without changing the animation?
Thank
[970x250.zip](https://github.com/airbnb/lottie-web/files/7482403/970x250.zip)
s!

**Comment by bodymovin** [comment]
hi, have you tried selecting the option "Remove expression properties (Reduces filesize)" on the "Expression options"  panel?

**Comment by parashut1986** [comment]
No. After I did it, it turned out 153kb. I'll try to remove a couple of minor elements, maybe this will help achieve the desired result. Thanks a lot!

**Comment by bodymovin** [comment]
How are you calculating that 153Kb? When I zip your files they only weight 108Kb
You should not include the data.json outside the banner folder

**Comment by parashut1986** [comment]
I understand correctly, this file outside the folder can be deleted?

**Comment by bodymovin** [comment]
yes, that's correct

**Comment by parashut1986** [comment]
Wow! Is it possible to somehow specify in the settings that this file is not duplicated? )

**Comment by bodymovin** [comment]
sorry, what do you mean by not duplicated?

**Comment by parashut1986** [comment]
When I press render, Bodymovin generates files just like in my archive. Those creates file data.json two times. Once in the banner folder, one outside. This is my settings
![2021-11-05_14-45-57](https://user-images.githubusercontent.com/93767439/140505742-b20c7d1b-ba2b-4474-b88d-893ffbef7a36.png)

**Comment by bodymovin** [comment]
I see. The other json is generated because you probably also have the Standard option selected, which produces a json.
I'll think of a way to clarify this, thanks

**Comment by Dr0mp** [comment]
If you use the banner export settings you can use cdn as a library origin instead of a local player, also zip file, and inline also works(that's to have it more compact)

If you use images (png' files) you can compress them into tiny png and under assets option you can tick the copy original files(i think?).

Too many keyframes will make the file heavy, if you stuff like wiggle expresions you might want to convert them to keyframe before the export and delete any excess.

**Comment by ldn24** [comment]
How did you change the Lottie library to “svg light” @parashut1986 @bodymovin

**Comment by Dr0mp** [comment]
> How did you change the Lottie library to “svg light” @parashut1986 @bodymovin

![image](https://user-images.githubusercontent.com/31309931/225497667-8356ddd8-bd79-403f-b0d3-9b030d76c055.png)

Activate banner from settings, and its there under the lottie library menu.

**Comment by ldn24** [comment]
I am using the bodymovin plugin. But my drop down only had the full-option.
